static/js/settings_users.js

```diff
@@ -114,14 +114,14 @@ function populate_users(realm_people_data) {
 
     var bots_table_html = "";
     _.each(bots, function (user) {
-        var bot_html = templates.render("admin_user_list", {user: user});
+        var bot_html = templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin});
         bots_table_html = bots_table_html.concat(bot_html);
     });
     bots_table.append(bots_table_html);
 
     _.each(active_users, function (user) {
         var activity_rendered;
-        var row = $(templates.render("admin_user_list", {user: user}));
+        var row = $(templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin}));
```

Comment:
@showell I am sorry if my words meant something else, but I never meant to say so. I'll try to rephrase my words

`admin_user_list.handlebars` displays the list of "Users" row wise and processes 1 user at a time.
Say "Users" has the following names
Othello
Iago
Zoe

For Administrator user(page_params.is_admin = true)


1. user = Othello , page_params.is_admin = true
display(user.name, user.email)
// If the logged in user is admin only then display the following buttons
if(page_params.is_admin == true)   
  if(user.is_admin == true) // Here we check if Othello is admin
    display(“Remove admin”)
  else
    display(“Make admin”)

2. User = Iago, page_params.is_admin = true
display(user.name, user.email)
// If the logged in user is admin only then display the following buttons
if(page_params.is_admin == true)
  if(user.is_admin == true) // Here we check if Iago is admin
    display(“Remove admin”)
  else
    display(“Make admin”)

3.User = Zoe, page_params.is_admin = true
display(user.name, user.email)
// If the logged in user is admin only then display the following buttons
if(page_params.is_admin == true) 
  if(user.is_admin == true) // Here we check if Zoe is admin
    display(“Remove admin”)
  else
    display(“Make admin”)

`page_params.is_admin` remains the same for a particular logged_in user. It also remains constant each time we are passing, but we need to pass it everytime as the template is getting processed line by line. That is why I have chosen the variable name to be `is_loggedin_user_admin` so that one does not confuse it with user.is_admin which refers to the user in the list.